css/index.css

```diff
@@ -1,3 +1,561 @@
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+	text-align: center;
+	margin: 0 20%;
+}
 
+@media( max-width: 800px ){
+	body {
+		width: 95%;
+		margin: 0 auto; 
+	}
+}
+
+@media ( max-width: 500px){
+	body{
+		width: 100%;
+	}
+}
+
+
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+
+h1 {
+	font-size: 50px;
+	letter-spacing: 1px;
+}
+
+h2 {
+	font-size: 12px;
+}
+
+@media( max-width: 800px ) {
+  h2 {
+    font-size: 16px;
+  }
+}
+
+h3 {
+	font-family: 'Titillium Web', sans-serif;
+	font-size: 12px;
+}
+
+@media( max-width: 800px ) {
+  h3 {
+    font-size: 16px;
+  }
+}
+
+.logo_800 {
+  margin-top: 30px;
+	height: 35px;
+	width: auto;
+	align-self: center; 
+	margin: 30px auto 20px auto;
+  margin-bottom: 20px;
+}
+
+@media( min-width: 800px ) {
+  .logo_800 {
+		display: none;
+  }
+}
+
+@media( max-width: 500px ) {
+  .logo_800 {
+		margin-top: 25px;
+  }
+}
+
+nav {
+	display: flex; 
+	justify-content: space-between;
+	align-content: center;
+	margin-top: 20px;
+}
+
+@media ( max-width: 800px){
+	nav {
+		width: 100%;
+		margin-top: 20px; 
+		flex-direction: column;
+		justify-content: center;
+		position: relative; 
+	}
+}
+
+@media ( max-width: 500px){
+	nav {
+		width: 100%;
+		margin-top: 0px; 
+		flex-direction: column;
+		justify-content: center;
+		position: relative; 
+	}
+}
+
+nav .menu-buttons a {
+	padding-left: 20px;
+	height: 20px; 
+	font-size: 10px;
+	text-decoration: none;
+	color: lightgrey;
+}
+
+@media( max-width: 800px ) {
+  nav .menu-buttons{
+		display: flex; 
+		justify-content: space-between; 
+		color: rgb(51, 51, 51);
+  }
+}
+
+@media( max-width: 500px ) {
+  nav .menu-buttons{
+		display: flex; 
+		flex-direction: column;
+		justify-content: space-between; 
+		color: rgb(51, 51, 51);
+		text-align: center;
+  }
+}
+
+@media( max-width: 800px ) {
+  nav .menu-buttons a{
+		font-size: 15px;
+		justify-content: space-between; 
+		width: 100%;
+	}
+}
+
+@media( max-width: 500px ) {
+  nav .menu-buttons a{
+		border-bottom: 1px solid darkgray;
+		height: 60px;
+		font-size: 20px;
+		width: 100%;
+		text-align: center; 
+		padding-top: 20px;
+  }
+}
+
+
+
+nav a:first-child{
+	padding-left: 0px;
+}
+
+@media( max-width: 500px ) {
+  nav .menu-buttons a:first-child{
+		border-bottom: 1px solid darkgray;
+		border-top: 1px solid darkgray;
+		height: 60px;
+		font-size: 20px;
+		width: 100%;
+		text-align: center; 
+		padding-top: 20px;
+		padding: auto;
```

Comment:
unfortunately `padding:auto` does not work. just give it a `padding-left` would be ideal.